Moving submission instructions to the website (#18)
naacl2022-reproducibility-track/naacl-utils: README.md

*Requires [Docker](https://www.docker.com/) and Python 3.7 or newer.* If you don't have a suitable Python installation, see [Installing Python](#installing-python) below.

> Instructions for using the tool should be kept in [the Dockerization tutorial](https://github.com/naacl2022-reproducibility-track/naacl2022-reproducibility-track.github.io/blob/main/_tutorial/04_submitting.md).
> Information relevant to developers of the tool should be kept in this Readme.

Reviewer: How about just:
```suggestion
Instructions for using the tool are in [the Dockerization tutorial](https://github.com/naacl2022-reproducibility-track/naacl2022-reproducibility-track.github.io/blob/main/_tutorial/04_submitting.md).
```

> ## Installing Python

Reviewer: We can probably take out this section as well, since we already have this info the CONTRIBUTING guide: https://github.com/naacl2022-reproducibility-track/naacl-utils/blob/moving-submission-instructions/CONTRIBUTING.md#making-a-pull-request
